core/src/test/java/tech/tablesaw/analytic/AnalyticQueryTest.java

```diff
@@ -81,10 +85,14 @@ public void toSqlStringNumbering() {
             .build();
 
     String expectd =
-        "SELECT\n"
-            + "RANK() OVER w1 AS myRank\n"
-            + "FROM myTable\n"
-            + "Window w1 AS (\n"
+        "SELECT"
+            + System.lineSeparator()
```

Comment:
this seems a little hard to read to me. Can we put each `System.lineSeparator()` on the end of the line instead of putting it on a new line?

Reply:
Yes, I don't like also. But it was done by the formatter plugin that is configurated in maven. 
In fact, it separated a lot of lines from the way I initially leave them.
Is it used?

Reply:
it wouldn't hurt to use a static import so you could just write ```lineSeparator()```. Even that is too verbose, but you can't fight Java names.

Reply:
You could make it even shorter by assigning it to a constant at the top of the file like `NEW_LINE`

One day we'll have Java 13's triple quotes and this will all be cleaner, but it's probably too early for that since Java 11 is the last LTS unless maybe we can just run the tests on Java 13, but compile with Java 11

Reply:
Yes I can use static import to reduce size, but about the new line I don't like it too.
But is what made the formatter plugin. I suppose that it sees a string concatenation that is two big and wants every concatenation in its new line.

What do you want me to do?
Only static import?
Reformat by hand trying to use fewer lines and ignoring formatter plugin? In that case are there any formatting rules that I must follow?

Reply:
You shouldn't format by hand. I'm not sure how the other options would look, but they're worth experimenting with

Reply:
Ok, then I should do just use a static import on all line separators? Any other thing it's needed to be changed?

Reply:
I would say we should try it and see how it looks

Reply:
At the end, I use a LINE_END constant that I've seen that was used in some places.